components/sysutils/neofetch/patches/01-fix-pkginfo.patch

```diff
@@ -0,0 +1,11 @@
+--- neofetch.1  2021-05-26 21:22:22.828639039
++++ neofetch    2021-05-26 21:22:10.350187007
+@@ -1585,6 +1585,9 @@
```

Comment:
Your Makefile settings and the patch file don't match.
You either need to create a patch file with paths (which I recommend) or
add PATCH_LEVEL= 0 to your Makefile.
Another thing: your copy's name of neofetch.1 with the unmodified original contents clashes with the man file that is part of the sources.
My recommendation is here to use another name. My usual workflow would be like this:
cd neofetch-7.1
cp -p neofetch neofetch.orig
vi neofetch   (make the changes)
cd ..
gdiff -u neofetch-7.1/neofetch.orig neofetch-7.1/neofetch >patches/01-fix-pkginfo.patch
(Using gdiff is not necessary; requirement is just a unified diff)

Reply:
Thanks, I've rebased and updated the patch file